docs/guides/deployment/kubernetes.md

```diff
@@ -10,9 +10,11 @@ search:
   boost: 2
 ---
 
-# Running flows with Kubernetes
+# Running flows with Kubernetes and a Kubernetes worker
```

Comment:
I think the original title still works. The guide was already using a worker, it just deferred to the Helm guide.

Reply:
cool